### DISCUSSION TITLE
Thank you for super cool project.  Possible to use cloud backends?

### DISCUSSION THREAD

**Discussion by 99bits** [basharovV/musicat#36] [Q&A]
Hi, thank you for this awesome project.
I have most of my media assets on Backblaze cloud because it's cheap and accessible anywhere. Would it be possible to connect to cloud backends?

**Comment by basharovV** [comment]
Hey, just wondering are you interested in having the app talk directly to Backblaze to stream tracks? Or use Rclone to mount it as a drive on your system? 

I'm also looking for a solution for this, I'd love to have the same music library on my desktop (for Musicat) and on my phone (any player I want). Currently I'm storing my music on Protondrive but the sync is buggy and on Android you need to manually download files from the Proton app to make them accessible to other apps...so I'm looking for an alternative. I'll look into having Backblaze as a "bring your own backend" solution

**Comment by 99bits** [comment]
I was thinking of Streaming, however to have both options would be great.
